modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/execution/executor/MethodExecutorGenerator.java

```diff
@@ -209,23 +221,25 @@ public ByteCodeAppender appender(Target implementationTarget) {
 
         ).make();
 
+    if (generatedByteCodeFile == null && LOGGER.isTraceEnabled()) {
```

Comment:
what's this for? Actually, if you look at usages, this parameter is just for testing as in any other situation it's always null .

Reply:
so that i can toggle this by setting the logger to TRACE